### PR TITLE
[RI] Release note for v1.7.1

### PR DESCRIPTION
**Pull request by slemeur**

### CODE REVIEW

File: content/ri/release-notes/v1.7.0.md [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
-Title: Release Notes for v1.7
+Title: RedisInsight Release Notes 1.7 
 date: 2020-09-10 00:00:00 +0000
 description: RediSearch 2.0 support and stability improvements
 weight: 95
 ---
 
+## 1.7.1
+
+Maintenance release for RedisInsight 1.7 including bug fixes and enhancements!
```
Comment (banker) [inline content/ri/release-notes/v1.7.0.md:10]: No need for "!".

File: content/ri/release-notes/v1.7.0.md [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+
+- Core:
+    - New public health-check API to make monitoring deployments easier.
+    - Display progress information during memory analysis.
+
+### Full details:
+
+Enhancements and bug fixes
+- Core:
+    - Fixed support for TLS Redis Cluster databases.
```
Comment (banker) [inline content/ri/release-notes/v1.7.0.md:22]: Fixed support for TLS in Redis Cluster databases.

File: content/ri/release-notes/v1.7.0.md [hunk 3]
```diff
@@ -1,10 +1,41 @@
...
+    - Application name is properly capitalized on Mac.
```
Comment (banker) [inline content/ri/release-notes/v1.7.0.md:23]: Suggest s/Mac/OSX

**Comment by bmansheim** [comment]
@banker Fixed in https://github.com/RedisLabs/redislabs-docs/commit/0af2bd0173302d367ad50a7d6111232d98af6220

**Pull request merged**